buildpack.toml

```
@@ -16,3 +16,8 @@ id = "heroku-20"
 
 [[stacks]]
 id = "heroku-22"
+
+[metadata]
+[metadata.release]
+[metadata.release.image]
+repository = "docker.io/heroku/buildpack-python"
```

Reading the TOML spec now (I always forget the TOML table nuanced), it seems that intermediate tables can be omitted when defining a child table. ie: It should be possible to omit eg the bare `[metadata]` table if we wanted:
https://toml.io/en/v1.0.0#table

Also, it's possible to write tables inline to make things less verbose:
https://toml.io/en/v1.0.0#inline-table

Combining those two, this should be equivalent:

```suggestion
[metadata.release]
image = { repository = "docker.io/heroku/buildpack-python" }
```

...and I think is a bit more concise - plus if we were ever to add more `metadata.release` options, they could be neatly grouped under `[metadata.release]` rather than needing a new `[metadata.release.<FOO>]` table for every new config option.